src/wp-includes/block-editor.php

```diff
@@ -361,6 +361,89 @@ function _wp_get_iframed_editor_assets() {
 	);
 }
 
+/**
+ * Finds the first occurrence of a specific block in an array of blocks.
+ *
+ * @since 6.3.0
+ *
+ * @param string $block_name Name of the block to find.
+ * @param array  $blocks     Array of blocks.
+ * @return array Found block, or empty array if none found.
+ */
+function wp_get_first_block( $block_name, $blocks ) {
+	foreach ( $blocks as $block ) {
+		if ( $block_name === $block['blockName'] ) {
+			return $block;
+		}
+		if ( ! empty( $block['innerBlocks'] ) ) {
+			$found_block = wp_get_first_block( $block_name, $block['innerBlocks'] );
+
+			if ( ! empty( $found_block ) ) {
+				return $found_block;
+			}
+		}
+	}
+
+	return array();
+}
+
+/**
+ * Adds styles and __experimentalFeatures to the block editor settings.
+ *
+ * @since 6.3.0
+ *
+ * @return array Post Content block attributes or empty array if they don't exist.
+ */
+function _wp_get_post_content_block_attributes() {
+	$is_block_theme = wp_is_block_theme();
+
+	global $post_ID;
```

Comment:
Missing docblock for global.

`* @global int $post_ID`

Reply:
Nit-pick: It would be good to follow the pattern of putting the `global` declaration to the very top of the function code.